### DISCUSSION TITLE
Donation Link

### DISCUSSION THREAD

**Discussion by m-dressler** [Project-OSRM/osrm-backend#6818] [Ideas]
I think it is so great that this is available as open-source software, but the fact that the API on https://project-osrm.org/ is also free while insanely fast is such a service to the internet, that I wanted to make at least a small donation. 

Since I couldn't find any links and I believe I wouldn't be the only one recognizing the value OSRM provides the world, I think it would be a good initiative to enable people to support this project.

**Comment by jcoupey** [comment]
If you mean to support the effort behind the demo server, then it probably makes sense to donate to FOSSGIS  (see https://github.com/Project-OSRM/osrm-backend/wiki/Demo-server).

Now if you mean donating to financially support dev work to maintain and enhance the project now and in the future to some extent, well there is no such thing right now. I'd say that is an open (and very interesting) question, especially as you mention since a lot of users are getting value out of OSRM.

**Comment by m-dressler** [comment]
I think both get their share. I'll definitely send some support to FOSSGIS but I think the efforts behind OSRM also could appreciate some support.

**Comment by DennisOSRM** [comment]
I used to have a buy-me-a-beer link, but since i stepped into a more quiet role the beer belongs to others these days. ☺️

**Comment by DennisOSRM** [comment]
We enabled GitHub sponsors for this. It should show a donate link at the top of the page.
